package.json

```diff
@@ -16,6 +16,10 @@
       "prettier --single-quote --write",
       "eslint --fix",
       "git add"
+    ],
+    "packages/useValidators/*.js": [
+      "prettier --single-quote --write",
+      "git add"
```

Comment:
Should we also run `eslint` on the validators package? Or are we putting that off until we can safely update the eslint packages here in the project root to understand the hooks api?

Reply:
I didn't add it because the package is so small. I didn't want to deal with the old eslint configurations and didn't want to bother trying to get it updated right now.